internal/services/compute/virtual_machine_scale_set_resource_test.go

```diff
@@ -4265,6 +4266,7 @@ resource "azurerm_virtual_machine_scale_set" "test" {
     type                       = "CustomScript"
     type_handler_version       = "2.0"
     auto_upgrade_minor_version = true
+    automatic_upgrade_enabled  = true
```

Comment:
unfortunately this resource is feature-frozen and as such unfortunately we'll need to roll this back:

```suggestion
```